lib/MusicBrainz/Server/Entity/Util/Application.pm

```diff
@@ -0,0 +1,29 @@
+package MusicBrainz::Server::Entity::Util::Application;
```

Comment:
The `Entity::Util` namespace seems a bit weird since it just contains a couple constants. Maybe they'd fit in lib/MusicBrainz/Server/Constants.pm with `OAUTH_*` prefixes?

Reply:
Yep, done in https://github.com/metabrainz/musicbrainz-server/pull/916/commits/8514b3e73781dbe9fde1f964f22f96ba53414a25